lib/WatcherProcess.php

```diff
@@ -393,6 +393,10 @@ private function generateWorkerCommand(Console $console): string {
             $parts[] = $ini;
         }
 
+        if ($console->isArgDefined("worker-args")) {
+            $parts = array_merge($parts, explode(' ', $console->getArg("worker-args")));
```

Comment:
Agree, not good approach. It will fail on any space in argument value. The only possible solution to parse it with Regexp. Seems CLImate doesn't support multiple argument. I can access to $argv directly, but it is dirty hack.

Do you have any plans to migrate to Symfony Console component?
It provides possibility to define multiple arguments: InputOption::VALUE_IS_ARRAY